samples/logging/stack-trace-cleaning/Core_8/SampleWithClean/SampleWithClean.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageReference Include="AsyncFriendlyStackTrace" Version="1.*" />
     <PackageReference Include="NLog" Version="4.*" />
     <PackageReference Include="NLog.Extensions.Logging" Version="1.7.4" />
-    <PackageReference Include="NServiceBus" Version="8.0.0-beta.1" />
-    <PackageReference Include="NServiceBus.Extensions.Logging" Version="2.0.0-beta.1" />
+    <PackageReference Include="NServiceBus" Version="[8.0.0-rc.1, 9.0.0)" />
+    <PackageReference Include="NServiceBus.Extensions.Hosting" Version="2.0.0-rc.1" />
```

Comment:
Same here, not sure why the package is changed

Reply:
Same as above